### DISCUSSION TITLE
Best choice for bounding object

### DISCUSSION THREAD

**Discussion by ShuffleWire** [cyberbotics/webots#4938] [Q&A]
I would be interested to know which type of Shape (as bounding object) is the best in order to get stable result from ODE, and reduce numerical instabilities, inaccuracy and so on...

I have a model on which I have several way to build the bounding object. It doesn't matter if the collision is not 100% realistic in term of shape, I focus first on the stability of my simulation.

For instance, I have to model a shaft as a Cylinder, but, I could also use a capsule Shape, because I intuitively think that avoiding the "rough" edges at the extremity could be beneficial, because I could have collision near to extremity with other object. 

For the same reason, I could use Box or Sphere to represent a mechanical stop, but I don't know what is best.

Any idea ?

**Comment by BenjaminDeleze** [comment]
You can have a look there to get a first idea: https://cyberbotics.com/doc/guide/tutorial-5-compound-solid-and-physics-attributes#how-to-choose-bounding-objects

**Reply by ShuffleWire** [reply]
Thanks, that is what I was looking for !
